### PR TITLE
Add posts limit parameter for each endpoint

### PR DESCRIPTION
**Pull request by ema987**
Using this parameter it will be possible to limit the number of posts fetched for each endpoint.
To keep backward compatibility, not using any limit will cause all the posts to be fetched

Hi @chrisj74,
thank you for the nice project! I had the same request as #2 so I added the feature myself and I hope you like it.

I saw rss2json has a parameter to limit the count of posts but using that would require you creating an account and getting an _apiKey_ from them. If you think it would be ok it won't be difficult to add another parameter for the apiKey to be provided as `data-rss2jsonapikey` and then instead of slicing the posts locally it would be possible to send the limit parameter to rss2json.

### CODE REVIEW

File: src/store/actions.js [hunk 1]
```diff
@@ -3,17 +3,26 @@ export function someAction (context) {
...
 */
 import axios from "axios";
-export function getPosts({ commit }, rssUrl) {
-  const rssArray = rssUrl.replace(/ /g, "").split(",");
-  rssArray.forEach(rss => {
+export function getPosts({ commit }, rss) {
+
+  const rssArray = rss.split(",");
+  const rssUrlsWithOptionsArray = rssArray.map(rss => {
+    const rssUrlWithOptions = rss.split(" ");
```
Comment (chrisj74) [inline src/store/actions.js]: Change to pipe `|` to avoid accidental use.

File: src/store/actions.js [hunk 2]
```diff
@@ -3,17 +3,26 @@ export function someAction (context) {
...
+    return {
+      rssUrl: rssUrlWithOptions[0],
+      rssItemsCount: rssUrlWithOptions[1]
```
Comment (chrisj74) [inline src/store/actions.js]: We should check this is a number or set to null.

Comment (ema987) [reply src/store/actions.js]: Please see if current implementation looks good!

**Comment by ema987** [comment]
Hi @chrisj74 , thank you for your reply!

I'm not very deep into js programming so I used a space because I saw it's common, like in lazy loading libraries, but I agree with you a pipe is much better!

I've made the changes you requested, let me know if more changes are needed!

**Pull request merged**